portfolio/src/main/webapp/index.html

```diff
@@ -2,45 +2,77 @@
+    <link rel="stylesheet" href="css/base/typography.css">
+    <link rel="stylesheet" href="css/base/utility.css">
+    <link rel="stylesheet" href="css/components/img-circle.css">
+    <link rel="stylesheet" href="css/sections/about.css">
+    <link rel="stylesheet" href="css/sections/header.css">
+    <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>Austin Teshuba</h1>
-
-      <img class="img-primary" src="/images/headshot.jpeg" alt="Profile Picture">
-
-      <p>
-          Hi! Welcome to my portfolio.
-
-      <a href="https://github.com/austinteshuba" target="_blank">Github</a>
-
-      <p> 
-          I'm a rising 3rd-year software engineering student at Western University.
-
-      <p>
-          I’m also a technologist knowledgeable in web development with Angular, 
-          iOS development with Swift, data analysis and ML with Python, algorithms, 
-          object oriented design, and much more.
 
-      <p>
-          Most importantly, I’m a curious person and a lightening fast learner. 
-          Whether I’m mastering web development to complete a pressing project at Deloitte, 
-          or figuring out Twilio and Microsoft Azure in 24 hours at MHacks, 
-          I always approach new problems with a positive attitude and relentless determination.
-
-      <p> 
-          If you're solving big problems with even bigger impacts, please reach out!
-          I'd love to chat more about how I can help.
-
-      <p>
-          What am I doing right now?
-      </p>
-
-      <button onclick="addActivity()">Let's find out</button>
-      <div id="activity-container"></div>
+  <body>
+    <div class="section-shadow-wrapper">
+      <header class="header-primary">
+        <div class="row u-text-shadow">
+          <div class="col-1-of-3 u-padding-sm header-container-left">
+            <div class="u-center u-text-center">
+              <img class="img-circle u-box-shadow" src="/images/headshot.jpeg" alt="Profile Picture">
+              <div class="caption-lg u-text-shadow">
+                Welcome to My Portfolio
+              </div>
+              <div class="icons">
+                <a href="https://www.linkedin.com/in/ateshub" target="_blank" >
+                  <img src="/images/linkedin.png" alt="LinkedIn Logo">
+                </a>
+                <a href="https://www.github.com/austinteshuba" target="_blank" >
+                  <img src="/images/github.png" alt="Github Logo">
+                </a>
+                <a href="https://www.devpost.com/teshuba" target="_blank" >
+                  <img src="/images/link.png" alt="Devpost Link">
+                </a>
+              </div>
+            </div>
+          </div>
+          <div class="col-2-of-3 u-text-right header-container-right">
```

Comment:
It would be helpful to insert some HTML comments to describe the main sections of the page. It takes some effort to infer this from the CSS class names.

Reply:
Agreed - added comments in confusing areas